backend/osmosewebsite/models/trap.py

```diff
@@ -0,0 +1,28 @@
+"""Trap-related models"""
+
+from django.db import models
+from django.contrib.postgres import fields
+from tinymce.models import HTMLField
+from .team_member import TeamMember
+
+
+class Trap(models.Model):
+    """
+    Table containing Trap will be used on the website
+    """
+
+    firstname = models.CharField(max_length=100, default="DefaultFirstName")
+    lastname = models.CharField(max_length=100, blank=True, null=True)
+    title = models.CharField(max_length=255, unique=True)
+    intro = models.CharField(max_length=255)
+    date = models.DateField(null=True, blank=True)
+    thumbnail = models.URLField(default="")
+    osmose_member_authors = models.ManyToManyField(TeamMember, blank=True)
+    linkedin_url = models.URLField(blank=True, null=True)
+    mail_address = models.EmailField(blank=True, null=True)
+    research_gate_url = models.URLField("Research Gate URL", blank=True, null=True)
+
+
+class Meta:
```

Comment:
With this indent, the Meta information doesn't have any use. The Meta class is used to specify how the model should be represented.

Reply:
The reprensentation of the model you set on this class Meta was good, you didn't have to remove it. Put it back on the Trap but with correct indent

Reply:
The indent is still incorrect, the Meta class should be inside the Trap class